src/ui/index.js

```diff
@@ -60,6 +60,19 @@ function buildGraph(data) {
     .force("nocollide", d3.forceCollide(data.nodeDistanceRatio * 200))
     .force("center", d3.forceCenter(width / 2, height / 2));
 
+  svg.append("defs").append("marker")
```

Comment:
could you wrap this in the `if (data.isIncludeBacklinks)` conditional as well. I don't think the arrows make too much sense unless you have the backlinks feature enabled

Reply:
Sure, added the conditional check